### PR TITLE
feat(authentication): implement a login modal for the gwr api

### PR DESCRIPTION
**Pull request by velrest**

### CODE REVIEW

**Comment by velrest** [comment]
Depends on #42

File: translations/de.yaml [hunk 1]
```diff
@@ -4,6 +4,9 @@ ember-gwr:
     building: Gebäude
     project: Projekt
 
+  authentication:
+    loginError: Bei der Anmeldung an der Housing-Stat Schnittstelle ist ein Fehler aufgetretten. Bitte versuchen Sie es nochmal.
```
Comment (czosel) [inline translations/de.yaml:8]:
```suggestion
    loginError: Bei der Anmeldung an der GWR-Schnittstelle ist ein Fehler aufgetretten. Bitte versuchen Sie es nochmal.
```

File: translations/de.yaml [hunk 2]
```diff
@@ -87,6 +90,12 @@ ember-gwr:
         extend: Erweiterte Filter einblenden
         collapse: Erweiterte Filter ausblenden
 
+    loginModal:
+      header: Bei Housing-Stat anmelden
```
Comment (czosel) [inline translations/de.yaml:94]:
```suggestion
      header: Im GWR anmelden
```

**Review by czosel — APPROVED** [review]
:+1:

**Pull request merged**